test/integration-test.js

```diff
@@ -1882,6 +1882,14 @@ vows.describe('integration tests')
       'space between ! and important': [
         'body{background-color:#fff  ! important}',
         'body{background-color:#fff!important}'
+      ],
+      'Capital letters': [
+        'a{color: red !ImporTant }',
+        'a{color:red!important}'
```

Comment:
Could you please check if mixed case `important` (like one above) works in browsers other than IE? It's OK if it does, otherwise we may mistakenly change styling.